yggdrasil_decision_forests/learner/distributed_gradient_boosted_trees/worker.cc

```diff
@@ -1110,7 +1110,7 @@ DistributedGradientBoostedTreesWorker::EvaluateWeakModelOnvalidationDataset() {
   // Schedule the prediction updates.
   utils::concurrency::StreamProcessor<int, int> processor(
       "update predictions", num_threads,
-      [num_examples, this, num_prediction_dimensions, &caches](
+      [num_examples, this, num_prediction_dimensions, &caches, batch_size](
```

Comment:
why this change?

Reply:
VS19 does not automatically inject consts.